src/proto/single/resp.rs

```diff
@@ -154,22 +157,24 @@ fn read<R: BufRead>(mut r: R) -> Result<RawResponse, Error> {
         }
         b'$' => {
             // Bulk String
-            // https://redis.io/topics/protocol#resp-bulk-strings
+            // https://redis.io/topics/error::protocol#resp-bulk-strings
```

Comment:
This looks like it may have been unintentional?

Reply:
Argh, good catch 💀